scripts/environment/prepare.sh

```diff
@@ -17,8 +17,8 @@ fi
 if ! cargo deny --version >& /dev/null ; then
   rustup run stable cargo install cargo-deny --force --locked
 fi
-if ! rust-license-tool --help >& /dev/null ; then
-  cargo install --git https://github.com/DataDog/rust-license-tool
+if ! dd-rust-license-tool --help >& /dev/null ; then
+  rustup run stable cargo install dd-rust-license-tool --version 1.0.1 --force --locked
```

Comment:
Do we need to lock down the version? We don't on other tools.

Reply:
We do actually lock down all of the others (see the lines above this one) except `cargo-deny`, which I'd argue we should be locking too.

Reply:
LOL took the line above to be exemplary. Fair enough.